nss-day-cohort-22/Class-Website
# added courtney's picture

## Description of Proposed Changes

-  Added Courtney's photos

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout addPic

```
Check to make sure courtney's photos are added

## Impacted Areas in Application

List general components of the application that this PR will affect:

*  Cohort.json

## Mentions @username

Tag users that need to review this code
@Chewieez @john-dulaney
